### ISSUE TITLE
[Feature]: Use the code signing certificate on the installer executable itself

### ISSUE THREAD

**Issue by roshkins** [VirtualDisplay/Virtual-Display-Driver#229]
### Is the feature related to a "problem"?

Since the driver code is now signed, it should be trivial to sign the installer executable itself, making it avoid the "Untrusted Program" dialog that appears.

### Your suggestion

Sign the installation executable with the driver key.

### Contact Details

Github: Roshkins

**Comment by itsmikethetech** [comment]
Hi Roshkins! Absolutely! The signing sponsor has a rule that everything signed has to be strictly MIT licensed, so we are just going through ensuring all installer components are licensed as MIT and available online, then we'll be getting it approved by SignPath before we sign the installer just to be safe.

**Comment by bud3699** [comment]
Yup! Don't worry we discussed probably everything you could imagine this morning, none stop everything! Fingers crossed we only move forward, the current installer isn't signed now because we set the release date for 24th for Christmas release, and as Mike said, we've emailed them about it too!

**Comment by roshkins** [comment]
Makes sense. I saw there was a gpl agreement.
